src/main/java/core/basesyntax/dao/csv/impl/CsvDaoImpl.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.dao.csv.impl;
+
+import core.basesyntax.dao.csv.CsvDao;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class CsvDaoImpl implements CsvDao {
+    private static final String WRITE_EXCEPTION = "Can`t write data to file";
+    private static final String CLOSE_BUFFERED_WRITER_EXCEPTION = "Can`t close BufferedWriter";
+    private static final String READ_FILE_PATH = "src/main/java/core/basesyntax/csv/database.csv";
+    private static final String WRITE_FILE_PATH = "src/main/java/core/basesyntax/csv/report.csv";
+
+    @Override
+    public List<String> readCsv() {
+        try {
+            return Files.readAllLines(Path.of(READ_FILE_PATH));
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```

Comment:
add message, include filepath which couldn't be read. Make debugging easier for users of your class